CHANGELOG-ABP.md

```diff
@@ -1,10 +1,15 @@
 # Release Notes
-## Chromium 68.0
+## Chromium 68.0 + ABP 0.4
```

Comment:
I think it makes sense to add ABP versioning as we're going to have 2 iterations - 1 for Chromium update, 2 for ABP update.

Reply:
@diegocarloslima Makes sense for you?

Reply:
Yes It does. This ABP version is the one that we were already placing in our tags, right? In the format of Chromium_ABP